staging/src/github.com/openshift/sdn/pkg/network/node/healthcheck.go

```diff
@@ -55,40 +67,36 @@ func runOVSHealthCheck(network, addr string, healthFn func() error) {
 		err = c.WaitForDisconnect()
 		utilruntime.HandleError(fmt.Errorf("SDN healthcheck disconnected from OVS server: %v", err))
```

Comment:
Maybe kill this line? Replace it with a `klog.Infof` after the reconnect indicating that OVS restarted but it's all cool.